uint8 dummy
---
uint16 barometerVal